StRoot/StETofUtil/StETofGeometry.cxx

```diff
@@ -60,16 +69,7 @@ StETofNode::StETofNode( const TGeoPhysicalNode& gpNode )
 {
 
     mGeoMatrix = static_cast< TGeoHMatrix* > ( gpNode.GetMatrix() );
-    /*
-    double* trans  = mGeoMatrix->GetTranslation();
-    double* rot    = mGeoMatrix->GetRotationMatrix();
 
-    LOG_INFO << trans[0] << "  " << trans[1] << "  " << trans[2] << endm;
-    
-    LOG_INFO << rot[0] << "  " << rot[1] << "  " << rot[2] << endm;
-    LOG_INFO << rot[3] << "  " << rot[4] << "  " << rot[5] << endm;
-    LOG_INFO << rot[6] << "  " << rot[7] << "  " << rot[8] << endm;
-    */
```

Comment:
Remove the following commented code as well https://github.com/star-bnl/star-sw/pull/358/files#diff-fa0ea941010f867b3171cb872336b971cf270aaa392bf777c57d206147fdc9d0R84-R93